### DISCUSSION TITLE
can i just use this project to build part of onlyoffice?

### DISCUSSION THREAD

**Discussion by alfabeto2019** [ONLYOFFICE/build_tools#347] [Q&A] [answered]
all i need is the sdkjs and web-apps part,so can i just run build_js.make() function?And cover this part of source codes for testing.

**Comment by ShockwaveNN** [comment, marked as answer]
I personally never done this, but I think if you dig code on python you can find everything you need

Also please look at [this code](https://github.com/ONLYOFFICE/build_tools/tree/develop/develop) maybe this helps

Moving this issue to discussion, since it's not an issue
